cmds/replay.c

```diff
@@ -583,8 +589,13 @@ void get_argspec_string(struct uftrace_task_reader *task,
 				}
 				print_args("\\\"");
 			}
-			else
+			else {
+				if (out_color == COLOR_ON)
+					print_args(TERM_COLOR_MAGENTA);
 				print_args("\"%.*s\"", slen + newline, str);
+				if (out_color == COLOR_ON)
+					print_args(TERM_COLOR_RESET);
```

Comment:
I suggest you add a new variable to save color for string (start and end) like args_comma and so on.  If you'd initialize them properly, the code will simply look like:

```
color_init() {
   if (COLOR_ON)
      string_start = TERM_COLOR_MAGENTA;
   else
      string_start = "";
   ...
}

get_argspec_string() {
   ...
   print_args("%s\"%.*s\"%s", string_start, slen + newline, str, start_end);
   ...
}
```

Reply:
I just updated this PR but didn't write it in a single line. I think single call is a bit more efficient, but it hurts readability, so made it as below:
```
get_argspec_string() {
   ...
   print_args("%s", string_start);
   print_args("\"%.*s\"", slen + newline, str);
   print_args("%s", start_end);
   ...
}
```